python/cm/bundle.py

```diff
@@ -258,7 +258,7 @@ def cook_roles(bundle):
             description=f'run action {name} of {act.prototype.type} {act.prototype.display_name}',
             category=[f'{bundle.name}'],
             bundle=bundle,
-            business_permit=False,
+            type='role',
```

Comment:
That's what I talked about - this should be like that:
`type = rbac.model.RoleTypes.role`
in order to not mistake or typo or to be able to find/rename every use of that constant

Reply:
Donein next commit